cortex/secondary/call_duration.py

```diff
@@ -2,14 +2,16 @@
 import numpy as np
 
 from ..feature_types import secondary_feature, log
-from ..raw.calls import calls
+from ..raw.telephony import telephony
 
 MS_IN_A_DAY = 86400000
+
+
 @secondary_feature(
     name='cortex.feature.call_duration',
-    dependencies=[calls]
+    dependencies=[telephony]
 )
-def call_duration(incoming=True, **kwargs):
+def call_duration(incoming, **kwargs):
```

Comment:
I think it may be a good idea to set the default value for `incoming` to 'incoming' to more closely match the original behavior of this feature - or perhaps to 'all'.

Additionally, it might be a good idea to do one of the following:
1. add an `isinstance` check to see if someone is trying to pass a boolean to `incoming` and use that to preserve the prior behavior (e.g. if they pass `True` return incoming calls, else return outgoing). This maintains more backwards compatibility.
2. Instead of changing the `incoming` variable in this way (as it becomes a bit unintuitive to have an `incoming` variable with the value `outgoing`), adding a variable like `call_direction`.